README.md

```diff
@@ -1,40 +1,119 @@
-# SPDX-Python-LicenseMatching
-Implement SPDX License Matching in Python. Project in CommunityBridge Linux Foundation 2020. 
-The Software Package Data Exchange (SPDX) specification is a standard format for communicating the components, licenses and copyrights associated with a software package. This Project is mainly based on implementing License Matching Techniques in Python. 
+# SPDX Python Library for LicenseMatching
+A Python Library to implement SPDX License Matching. This Project is a result of CommunityBridge Linux Foundation 2020 contribution by [@anshuldutt21](https://github.com/anshuldutt21/).
 
-## Check if 2 License Texts match
-License Texts are matched based on the [SPDX License Matching Guidelines](https://spdx.dev/license-list/matching-guidelines/). In order to implement License Matching between 2 License Texts, do the following steps.
+Home: https://github.com/anshuldutt21/spdx_python_licensematching/
+
+Issues: https://github.com/anshuldutt21/spdx_python_licensematching/issues
+
+# License
+[Apache-2.0](https://github.com/spdx/tools-python/blob/master/LICENSE)
+
+# Features
+
+*    An Interface which will check text against a license template using the license matching guidelines.
+*    An Interface which will check text and return all matching SPDX listed license ID's.
+*    An interface which takes 2 license texts as input and returns a boolean indicating if the 2 licenses match per the license matching guidelines.
+*    When there is not a match, a return value provided making it possible to describe where and why the license does not match.
 
-Clone the Repository and move to the spdx-python-licensematching folder.
+# Installation
 
-Move the files in the normalize_license_text folder.
+As always you should work in a virtualenv or venv. Do the following steps to setup the Python Package in your system.
 
-Execute the command 
 ```bash
-Python normalize.py <File1> <File2>
+git clone https://github.com/anshuldutt21/spdx_python_licensematching.git
+cd spdx_python_licensematching
+pip install -e .
 ```
-## Compare a License and a Template
-License templates are implemented on the basis of the [SPDX License Matching Guidelines](https://spdx.dev/license-list/matching-guidelines/). In order to use the feature  of comparing a license text and a template-
+
+# How To Use
+
+## Match 2 License Texts
+License Texts are matched based on the [SPDX License Matching Guidelines](https://spdx.dev/license-list/matching-guidelines/). In order to implement License Matching between 2 License Texts, do the following steps.
+
+Input all the Text Files you want to match at the `input_text_files` directory inside `normalize_license_text`. This can be done by the following steps.
 
 ```bash
-cd compare_template_text/
+mv <TextFile_A> normalize_license_text/input_text_files 
+mv <TextFile_B> normalize_license_text/input_text_files 
```

Comment:
I don't think we need these moving files. Also, it should be cp instead of mv, since user might lose the file from the original location.